tracing-subscriber/src/filter/env/field.rs

```diff
@@ -289,6 +316,79 @@ impl Ord for MatchPattern {
     }
 }
 
+// === impl MatchDebug ===
+
+impl MatchDebug {
```

Comment:
nit: i'd maybe add a one sentence doc comment explaining how this used.

(i know what it does, but others might not!)